nvm.sh

```diff
@@ -645,16 +645,16 @@ nvm_print_formatted_alias() {
   DEST_FORMAT='%s'
   VERSION_FORMAT='%s'
   local NEWLINE
-  NEWLINE="\n"
+  NEWLINE="\\n"
```

Comment:
This just doesn't seem right to me. The intention is to insert a literal newline; not to literally add a backslash and an `n`

Reply:
Yes, it's to add a new line, that's the tricky point.

Reply:
Yes, it's newline, that's the tricky point I think.

Reply:
`\\n` isn't a newline tho :-/

Reply:
Really? I tested it with this code block and it looks fine:
```sh
export NEWLINE="\n"
command printf -- "${NEWLINE}"
unset NEWLINE
export NEWLINE="\\n"
command printf -- "${NEWLINE}"
unset NEWLINE
command printf -- "${NEWLINE}"
```

Reply:
I'm confused; does that mean `\n` and `\\n` are the same? Or is the former a literal newline, and the latter a literal `\n` that the shell later turns into a newline?

Reply:
Yeah, I think they are the same as the wiki mentions, the old usage may be wrong, and just fallback as what we want, that's the reason why they output the same.

Reply:
so on this line; can it be replaced with `'\n'`?